homeassistant/components/nws/const.py

```diff
@@ -35,7 +35,7 @@
         "Hot",
         "Cold",
     ],
-    ATTR_CONDITION_SNOWY: ["Snow", "Sleet", "Blizzard"],
+    ATTR_CONDITION_SNOWY: ["Snow", "Sleet", "Snow/sleet", "Blizzard"],
```

Comment:
It would be cleaner if these were all constants but thats a bit of big ask for a bugfix.

Reply:
Even more ideal if they were constants from upstream library